### PR TITLE
style & android improvements

### PR DESCRIPTION
**Pull request by fatihtoker**

### CODE REVIEW

File: AppStyle.js [hunk 1]
```diff
@@ -1,4 +1,17 @@
 import { StyleSheet } from 'react-native';
+import Colors from './constants/Colors';
+
+const defaultButton = {
+    fontSize: 20,
+    color: 'white',
```
Comment (barhantas) [inline AppStyle.js:6]: Colors.white

File: AppStyle.js [hunk 2]
```diff
@@ -11,18 +24,17 @@ export default StyleSheet.create({
         backgroundColor: "#fff"
```
Comment (barhantas) [inline AppStyle.js:24]: Colors.white

File: AppStyle.js [hunk 3]
```diff
@@ -11,18 +24,17 @@ export default StyleSheet.create({
...
     },
     defaultButton: {
```
Comment (barhantas) [inline AppStyle.js:26]: `defaultButton,` is enough

File: AppStyle.js [hunk 4]
```diff
@@ -11,18 +24,17 @@ export default StyleSheet.create({
...
-        overflow: 'hidden'
+       ...defaultButton
     },
     defaultButtonContainer: {
         alignSelf: 'center',
         justifyContent: 'center'
+    },
+    outlinedButton: {
+        ...defaultButton,
+        backgroundColor: '#fff',
```
Comment (barhantas) [inline AppStyle.js:35]: Colors.white

File: AppStyle.js [hunk 5]
```diff
@@ -11,18 +24,17 @@ export default StyleSheet.create({
...
+        color: Colors.green500,
+        borderWidth: 1,
+        borderColor: Colors.green500
     }
 });
```
Comment (barhantas) [inline AppStyle.js:40]: end of file

File: screens/DateSelectorScreen.js [hunk 6]
```diff
@@ -2,45 +2,39 @@ import React, { useState, useEffect } from "react";
 import DateTimePicker from "@react-native-community/datetimepicker";
 import AppStyle from "../AppStyle";
 
-import { StyleSheet, Text, View } from "react-native";
+import { AsyncStorage, StyleSheet, Text, View, Platform, Image } from "react-native";
```
Comment (barhantas) [inline screens/DateSelectorScreen.js:5]: AsyncStorage is not used

File: screens/DateSelectorScreen.js [hunk 7]
```diff
@@ -49,8 +43,30 @@ export default function DateSelectorScreen({ navigation, route }) {
         is24Hour={true}
         display="default"
         onChange={onChange}
-      />
+      />}
+
+      {Platform.OS === 'android' &&
+
```
Comment (barhantas) [inline screens/DateSelectorScreen.js:49]: empty line

File: screens/DateSelectorScreen.js [hunk 8]
```diff
@@ -49,8 +43,30 @@ export default function DateSelectorScreen({ navigation, route }) {
...
+        <Image
+          style={styles.image}
+          source={require('../assets/images/date-selector.png')}
+        />
+      }
       <View style={styles.buttonContainer}>
+        {Platform.OS === 'android' && !dateTimePickerVisible &&
+
```
Comment (barhantas) [inline screens/DateSelectorScreen.js:57]: empty line

File: screens/DateSelectorScreen.js [hunk 9]
```diff
@@ -49,8 +43,30 @@ export default function DateSelectorScreen({ navigation, route }) {
...
+          <View style={{ marginBottom: 20 }}>
```
Comment (barhantas) [inline screens/DateSelectorScreen.js:58]: move it to stylesheet

File: screens/HomeScreen.js [hunk 10]
```diff
@@ -86,22 +88,16 @@ export default function HomeScreen({ navigation, route }) {
...
-          onPress={async () => {
-            await removeFromStorage("quarantineStartDate");
-            await removeFromStorage("quarantineDurationInDays");
-            navigation.navigate("DateSelector");
-            navigation.reset({
-              index: 0,
-              routes: [{ name: "DateSelector" }]
-            });
-          }}
+          onPress={onFinishQuarantinePressed}
```
Comment (barhantas) [inline screens/HomeScreen.js:94]: onPressFinishQuarantine may be better

File: screens/TimeSelectorScreen.js [hunk 11]
```diff
@@ -1,72 +1,94 @@
...
+import Colors from "../constants/Colors";
 
 export default function TimeSelectorScreen({
   navigation,
   route: { params: { selectedStartDate } } = {}
 }) {
-  const [day, setDay] = useState("14");
 
-  useEffect(() => {}, []);
+  useEffect(() => { }, []);
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:19]: delete it

File: screens/TimeSelectorScreen.js [hunk 12]
```diff
@@ -1,72 +1,94 @@
...
+            })}
+
+          </Picker>
+        </View>
+        <View style={Platform.OS === 'ios' ? styles.buttonContainer : {}}>
+          <Button
+            style={styles.button}
+            onPress={async () => {
+              await writeToStorage("quarantineDurationInDays", day);
+              // utilizeStartDate(JSON.parse(date));
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:71]: delete the commet pls

File: screens/TimeSelectorScreen.js [hunk 13]
```diff
@@ -1,72 +1,94 @@
...
+          style={styles.image}
+          source={require('../assets/images/time-selector.png')}
+        />
+      }
+      <View style={Platform.OS === 'android' ? styles.buttonContainer : { flex: 1 }}>
+        <View style={Platform.OS === 'android' ? styles.pickerWrapper : {}}>
+          <Picker
+            selectedValue={day}
+            onValueChange={(val) => { setDay(val) }}
+            style={Platform.OS === 'android' ? styles.picker : {}}
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:55]: `Platform.OS === 'android' && styles.picker`

File: screens/TimeSelectorScreen.js [hunk 14]
```diff
@@ -1,72 +1,94 @@
...
+            prompt="Duration"
+          >
+            {days.map((item, index) => {
+              return (
+                <Picker.Item label={item.toString()} value={item.toString()} key={item} />
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:60]: declaret on top once -->item.toString()

File: screens/TimeSelectorScreen.js [hunk 15]
```diff
@@ -1,72 +1,94 @@
...
+              );
+            })}
+
+          </Picker>
+        </View>
+        <View style={Platform.OS === 'ios' ? styles.buttonContainer : {}}>
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:66]: `Platform.OS === 'ios' ? styles.buttonContainer : {}`
insted of this you can use like;

`condition && yourVariable`

File: screens/TimeSelectorScreen.js [hunk 16]
```diff
@@ -1,72 +1,94 @@
...
+            onPress={async () => {
+              await writeToStorage("quarantineDurationInDays", day);
+              // utilizeStartDate(JSON.parse(date));
+              await writeToStorage("quarantineStartDate", utilizeStartDate(JSON.parse(selectedStartDate)));
+              await writeToStorage("tasks", TASKS.slice(0, day));
+              navigation.reset({
+                index: 0,
+                routes: [{ name: 'App' }],
+              });
+              navigation.navigate("App");
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:78]: this line was deleted why did you put it again ?

File: screens/TimeSelectorScreen.js [hunk 17]
```diff
@@ -76,6 +98,24 @@ const styles = StyleSheet.create({
...
+    ...AppStyle.outlinedButton,
+    padding: 0,
+    marginBottom: 20,
+    alignItems: 'stretch',
+    justifyContent: 'flex-start'
+
+  },
+  image: {
+    flex: 1,
+    width: undefined,
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:115]: is this necessary to sen 'undefined' to width and height

File: screens/DateSelectorScreen.js [hunk 18]
```diff
@@ -2,45 +2,39 @@ import React, { useState, useEffect } from "react";
...
 
 import { readFromStorage } from "../utils";
+import moment from "moment";
+import Colors from "../constants/Colors";
 
 export default function DateSelectorScreen({ navigation, route }) {
   const [date, setDate] = useState(new Date());
+  const [dateTimePickerVisible, setDateTimePickerVisible] = useState(false);
+
+  useEffect(() => {
```
Comment (barhantas) [inline screens/DateSelectorScreen.js:16]: we can delete it

File: screens/TimeSelectorScreen.js [hunk 19]
```diff
@@ -1,72 +1,94 @@
...
+  console.log('selectedDate: ', selectedStartDate);
+
+  const [day, setDay] = useState('14');
+
+
+
 
   const days = [];
 
   for (var i = 1; i <= 30; i++) {
```
Comment (barhantas) [inline screens/TimeSelectorScreen.js:35]: how many times this for loop gonna work ?

File: screens/TimeSelectorScreen.js (hunk 17 above)
Comment (fatihtoker) [reply screens/TimeSelectorScreen.js:115]: yes, using 'flex: 1' and resizeMode: 'contain' sets image's size to fill remaining space but you have to unset image's original size to make it work

File: screens/HomeScreen.js (hunk 10 above)
Comment (barhantas) [reply screens/HomeScreen.js:94]: .

**Pull request merged**